gormsupport/cleaner/db_clean.go

```diff
@@ -39,18 +38,22 @@ import (
 func DeleteCreatedEntities(db *gorm.DB) func() {
 	hookName := "mighti:record"
 	type entity struct {
-		table   string
-		keyname string
-		key     interface{}
+		table string
+		keys  map[string]interface{}
 	}
 	var entires []entity
 	hookRegistered := db.Callback().Create().Get(hookName) != nil
 	if hookRegistered {
 		hookName += "-" + uuid.NewV4().String()
 	}
 	db.Callback().Create().After("gorm:create").Register(hookName, func(scope *gorm.Scope) {
-		log.Logger().Debugln(fmt.Sprintf("Inserted entities from %s with %s=%v", scope.TableName(), scope.PrimaryKey(), scope.PrimaryKeyValue()))
-		entires = append(entires, entity{table: scope.TableName(), keyname: scope.PrimaryKey(), key: scope.PrimaryKeyValue()})
+		fields := scope.PrimaryFields()
+		keys := make(map[string]interface{})
+		for _, field := range fields {
+			keys[field.DBName] = field.Field.Interface()
+		}
+		log.Logger().Debugln(fmt.Sprintf("Inserted entities from %s with keys %v", scope.TableName(), keys))
+		entires = append(entires, entity{table: scope.TableName(), keys: keys})
```

Comment:
should this var be "entities" ?

Reply:
I didn't rename the original var. But we can rename it now.

Reply:
I would rename it, this seems to be a typo.

Reply:
Done.